Mapping Tools/views/Sliderator/SlideratorView.xaml.cs

```diff
@@ -84,11 +84,8 @@ private void Import_Slider(object sender, RoutedEventArgs e)
                     Beatmap beatmap = editor.Beatmap;
                     Timing timing = beatmap.BeatmapTiming;
                     List<HitObject> markedObjects = selected;
-                if (markedObjects.Count > 1 || markedObjects.Count < 1)
-                    continue;
-                if (!markedObjects.ToArray()[0].IsSlider)
-                    continue;
-                GraphHitObjectElement.HitObject = markedObjects.ToArray()[0];
+                    
+                    GraphHitObjectElement.HitObject = markedObjects.Find(s => s.IsSlider);
```

Comment:
This is better but I think it will throw an exception if it cant find any. If there are no sliders selected i guess just dont change hit object.

Reply:
Created Trycatch exception for ArugmentNull